CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Alfred API - Changelog
 
+## 2.2.1 (Unreleased)
```

Comment:
According to https://xenitsupport.jira.com/wiki/spaces/XEN/pages/411205669/XEP-7+Versioning+schema you should increment minor, not micro.